src/viam/services/motion/client.py

```diff
@@ -121,6 +139,24 @@ async def move_on_globe(
 
         You can monitor the progress of the ``move_on_globe()`` call by querying ``get_plan()`` and ``list_plan_statuses()``.
 
+        ::
+
+            motion = MotionClient.from_robot(robot=robot, name="builtin")
+
+            # Get the ResourceNames of the base and movement sensor
+            my_base_resource_name = Base.get_resource_name("my_base")
+            mvmnt_sensor_resource_name = MovementSensor.get_resource_name(
+                "my_movement_sensor")
+            #  Define a destination GeoPoint at the GPS coordinates [0, 0]
+            my_destination = movement_sensor.GeoPoint(latitude=0, longitude=0)
+
+            # Move the base component to the designated geographic location, as reported by
+            # the movement sensor
```

Comment:
I think the max line length is 120 or 140 so can merge these too:
```suggestion
            # Move the base component to the designated geographic location, as reported by the movement sensor
```